lib/src/builder/codegen/typed_map_impl_generator.dart

```diff
@@ -369,6 +379,9 @@ class _TypedMapImplGenerator extends TypedMapImplGenerator {
         factoryNames =
             declaration.factories.map((factory) => FactoryNames(factory.name.name)).toList(),
         member = declaration.props.either,
+        allPropsMixins = declaration.props.either.nodeHelper.mixins
+            ?.map<Identifier>((mixin) => mixin.name)
+            ?.toList(),
```

Comment:
I think this implementation might not be correct when using the shorthand syntax, since `.nodeHelper.mixins` on a mixin will be empty.

For class components, we're using `allPropsMixins`, which does account for that case. Could we share that same implementation between `ClassComponentDeclaration` and `PropsMapViewOrFunctionComponentDeclaration` so that we can use it here as well?

Reply:
Definitely, good catch. Updated in ea0e4aa!